### ISSUE TITLE
GNU "… or later" licenses are missing

### ISSUE THREAD

**Issue by htgoebel** [github/choosealicense.com#713]
Please add the "… or later" variant for all GNU licenses. Effected are at least
* AGPL 3.0 https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/agpl-3.0.txt#L605
* GPL 2.0 https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/gpl-2.0.txt#L279
* GPL 3.0 https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/gpl-3.0.txt#L610
* LGPL 2.1 https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/lgpl-2.1.txt#L453
* LGPL 3.0 https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/lgpl-3.0.txt#L186

##### Background
Some of my repos at github are displayed as "GPL-3.0", but it is "GPL-3.0 or later"., e.g. https://github.com/debops/yaml2rst/. I've been told by github support team to raise the issue here:
> It's my understanding that the only license names that will be displayed are those listed on http://choosealicense.com, so it's expected that you'd see GPL-3.0 and not GPL-3.0 or later on these repositories.
> If you wish to suggest a change you can do so via the choosealicense repository:

**Comment by mlinksva** [comment]
Adding duplicate license texts here won't help github.com disambiguate "only", "or-later", or unspecified applications of GPL family licenses -- there is no difference in the license text.

I see that you have specified or later in source code headers eg https://github.com/debops/yaml2rst/blob/39ac2ed3c055a2eda5438117c5ae54b7be559919/yaml2rst/__main__.py#L12 and package metadata https://github.com/debops/yaml2rst/blob/39ac2ed3c055a2eda5438117c5ae54b7be559919/setup.py#L38-L39) -- very good. github.com doesn't presently look outside license files for additional statement like these which might be used to refine license detection such as GPL-3.0 (which of course is true; or later is an option a licensee can choose to take up or not) to GPL-3.0-or-later. That can't be addressed in this repo.

In summary, https://github.com/licensee/licensee is the code github.com uses to detect licenses. It optionally looks at package metadata but currently isn't aware of "or later" refinements. That'd need to be addressed there. Finally, github.com would need to utilize the option. Closing this issue as there's nothing that's actionable in this repo. Thanks for opening it though!

**Comment by sema0703** [comment]
> Please add the "… or later" variant for all GNU licenses. Effected are at least
> * AGPL 3.0 https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/agpl-3.0.txt#L605
> * GPL 2.0 https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/gpl-2.0.txt#L279
> * GPL 3.0 https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/gpl-3.0.txt#L610
> * LGPL 2.1 https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/lgpl-2.1.txt#L453
> * LGPL 3.0 https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/lgpl-3.0.txt#L186
> 
> ##### Background
> Some of my repos at github are displayed as "GPL-3.0", but it is "GPL-3.0 or later"., e.g. https://github.com/debops/yaml2rst/. I've been told by github support team to raise the issue here:
> > It's my understanding that the only license names that will be displayed are those listed on http://choosealicense.com, so it's expected that you'd see GPL-3.0 and not GPL-3.0 or later on these repositories.
> > If you wish to suggest a change you can do so via the choosealicense repository:
> 
>

**Comment by sema0703** [comment]
https://github.com/github/choosealicense.com/blob/spdx-license-templates/_data/fields.yml
